### ISSUE TITLE
A11y_VscodeWebviewUiToolkit_AllComponents_Combobox_ScreenReader: The NVDA does not announces the names of the menu items like "option label#1" when navigating using down arrow key.

### ISSUE THREAD

**Issue by Shivamd1** [microsoft/vscode-webview-ui-toolkit#181]
### Actual:
The NVDA does not announces the names of the menu items like "option label#1" when navigating using down arrow key. It only announces "blank" for all the menu items

Similar issue is observed with Jaws

Similar issue is with the "Dropdowns" scenario

### Expected:
The NVDA should be announcing the names of the menu items like "option label#1" along with menu item numbering like 1 of 3 when navigating using down arrow key. 

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#SH_VscodeWebviewUiToolkit_Aug2021;#Webview UI Toolkit for Visual Studio Code;#Win32;#FTP;#DesktopApp;#WCAG1.3.1;#NVDA;#Jaws;#STP;#AINotInScope;

### Environment Details:
Application Name: Vscode Webview Ui Toolkit
Windows Version: Win10
Screen Reader: NVDA(2021.1), Jaws(2021.2102.34)

### Repro Steps:

1. Launch VS Code
2. Open WebView UI Toolkit: All Component extension
3. Tab to Combo box element and tab to first combo box and expand it
4. Start NVDA use down arrow to navigate and check the announcement

### User Impact:
Screen Reader users will not get the proper information about the menu items in comboboxes

[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**Comment by chrisdholt** [comment]
> Thanks for letting us know @yannickowow. We will investigate and report back.
> 
> @chrisdholt does this sound familiar at all?

I think this is an NVDA bug - I’ll dig in a bit today to confirm.

**Comment by Shivamd1** [comment]
@hawkticehurst  Could you please provide the latest vsix file so that we can verify it, also could you please reopen the bug, it should only be closed by trusted tester

**Comment by Roopesh-T** [comment]
@hawkticehurst @daviddossett Could you please provide the latest vsix file so that we can verify it, also could you please reopen the bug, it should only be closed by trusted tester

**Comment by daviddossett** [comment]
@Roopesh-T we actually removed this code from the library so it's no longer applicable for a11y testing.

**Comment by Roopesh-T** [comment]
As per the above comment, the code is removed from the library, so closing the bug.

![image](https://user-images.githubusercontent.com/86601127/218697826-ca45f5c1-749f-4de5-8e69-563d6683066d.png)

![image](https://user-images.githubusercontent.com/86601127/218698114-d979eae5-966c-44dc-9416-30042f586d2e.png)
